components/Nav.vue

```diff
@@ -13,12 +13,12 @@
           v-if="$auth.loggedIn"
           class="flex justify-center content-center flex-col"
         >
-          <div class="flex flex-row">
-            <span class="text-base">Hi, {{ $auth.user.name }}</span>
+          <div class="flex flex-row text-gray-600">
+            <span class="text-base p-1">Hi, <span class="font-semibold text-red-700">{{ $auth.user.name }}</span></span>
```

Comment:
You seem to have an extra span

Reply:
That's to color the username differently

Reply:
Ah, I'm blind